src/components/modules/banners/ausMoment/AusBanner.tsx

```diff
@@ -0,0 +1,241 @@
+import React from 'react';
+import { css } from '@emotion/core';
+import { Hide, Container, Columns, Column } from '@guardian/src-layout';
+import { space } from '@guardian/src-foundations';
+import { from } from '@guardian/src-foundations/mq';
+import AusBannerHeader from './components/AusBannerHeader';
+import AusBannerBody from './components/AusBannerBody';
+import AusBannerCtas from './components/AusBannerCtas';
+import AusBannerCloseButton from './components/AusBannerCloseButton';
+import {
+    AusBannerTicker,
+    CurrentSupporterNumber,
+    GoalSupporterNumber,
+} from './components/AusBannerTicker';
+import AusBannerAnimation from './components/AusBannerAnimation';
+import { BannerRenderProps } from '../common/types';
+import { bannerWrapper, validatedBannerWrapper } from '../common/BannerWrapper';
+import { brand } from '@guardian/src-foundations';
+
+// -- styles -- //
+const containerStyles = css`
+    position: relative;
+    background: ${brand[400]};
+    display: flex;
+    flex-direction: column;
+    border-top: 1px solid white;
```

Comment:
should this be blue?

Reply:
I think it's from an old code, I'll remove it!

Reply:
the design does have a light blue top border